tools/SetupFlow/DevHome.SetupFlow/Views/EditClonePathDialog.xaml

```diff
@@ -47,6 +47,11 @@
                                 <SolidColorBrush Color="{ThemeResource SystemAccentColor}"/>
                             </Setter.Value>
                         </Setter>
+                        <Setter Property="Foreground">

```

Comment:
Do you get all this button styling (foreground and background) for free if you use the ContentDialog's PrimaryButton and Secondary button? "White" isn't going to fix the accessibility issue if someone has a particuarly light theme.

Reply:
Done.  For some reason, when I first tested this, the foreground was blue.  Now it is white, like the AddRepo dialog.

Reply:
Never mind, clearly there's details I don't know since it seems you're doing some pretty complex theme setting depending on dev drive stuff. (Still think it should be a theme brush but I can't figure out which one right now)